test/WebJobs.Script.Tests.Integration/EndToEndTestFixture.cs

```diff
@@ -159,6 +162,56 @@ public virtual void Dispose()
             Host.Stop();
             Host.Dispose();
             ServiceBusQueueClient.Close();
+            if (DocumentClient != null)
```

Comment:
You should be able to do `DocumentClient?.Dispose()` here. Saves a few lines :-)